nginx/nginx.conf

```diff
@@ -0,0 +1,15 @@
+events {
+
+}
+
+http {
+        error_log /etc/nginx/error_log.log warn;
+
+        server {
+                listen 80;

```

Comment:
i propose we make this 8080 o something else than port 80 so it does not easily conflict with othe rservices or conflicts with rights required for ports lower than 1000

Reply:
I will change it in the compose file. This is for internal nginx container so it does not matter in any way.

Reply:
done, will be `8088`.